gm4_shiny_items/data/gm4_shiny_items/functions/blacklist_items.mcfunction

```diff
@@ -0,0 +1,125 @@
+# Run from Main
+# @s = All items within 30 blocks of a player
+
+# TO BE CHANGED
+execute if entity @s[tag=!gm4_shiny_blacklist] unless entity @s[nbt={Item:{Count:1b}}] run tag @s add gm4_shiny_blacklist
+
+### ITEMS THAT COULD CAUSE PROBLEMS ###
+# Could cause player to get item with tag "unstackable:1b"
+execute if entity @s[tag=!gm4_shiny_blacklist] if block ~ ~-.6 ~ hopper run tag @s add gm4_shiny_blacklist
+
+### ITEMS THAT ALREADY HAVE ENCHANTMENT GLOW ###
+# Enchanted Items
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{tag:{Enchantments:[{}]}}}] run tag @s add gm4_shiny_blacklist
+# Misc (1)
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:enchanted_book"}}] run tag @s add gm4_shiny_blacklist
+# Bottles (4)
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:potion"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:splash_potion"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:lingering_potion"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:experience_bottle"}}] run tag @s add gm4_shiny_blacklist
+
+### ITEMS THAT CAN BE ENCHANTED ALREADY ###
```

Comment:
I don't think this section (Items that can be enchanted already) is necessary. What if a player for some reason want a shiny item without an actual enchantment? Removing this section would greatly reduce the number of commands to be run when this function is ran